scripts/run_performance_tests.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
```

Comment:
You have two performance test shell files? Please remove the duplicate. I think you only need this one.